sweddit_combined/events/warringchina.txt

```diff
@@ -16,12 +16,6 @@ country_event = {
 	immediate = {
 		hidden_effect = {
 			set_country_flag = claims_on_china
-			ROOT = {
```

Comment:
What is the effect of removing this? Will the claims still be set but without any pop-up?

Reply:
This is the claim on all of china event, and it triggers the generate leader event which has now been removed. Thought it might cause issues when we removed the generate leader event so I took out the trigger.